frame/error.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/** @fileoverview TODO */
+
+/** TODO */
+export function errorMessage(error: unknown): string {
+  return error instanceof Error
+    ? error.stack || error.toString()
+    : JSON.stringify(error) ?? "undefined";
```

Comment:
Operator precedence here would be clearer with some parens

Reply:
I agree, but unfortunately Prettier took them out. I guess it thinks they're not needed because the line breaks are meant to show precedence.